pulsar-flink-connector/src/main/java/org/apache/flink/streaming/connectors/pulsar/internal/PulsarMetadataReader.java

```diff
@@ -66,6 +66,9 @@
 @Slf4j
 public class PulsarMetadataReader implements AutoCloseable {
 
+    // system topics are not filtered out by default in Pulsar 2.10.0
+    private static final String SYSTEM_TOPIC_PREFIX = "__";
```

Comment:
https://github.com/streamnative/flink/issues/84 There is one exceptional system topic that does not start with __. Need to figure out the best way to filter system topics

Reply:
What about the system topic not start with `__`?

Reply:
2.10.1 there will be a class `SystemTopicNames` allow us to filter system topics defined by Pulsar. However currently we can onlyimplement our own method to include all the system topics, and upgrade once 2.10.1 is out.

Reply:
I found that the one system topic `transaction_coordinator_assign` not starting with __ resides in system namespace, so it is ok we skip it.